portfolio/src/main/webapp/organizereventedit.html

```diff
@@ -24,10 +27,88 @@ <h1 id="eventSaved"></h1>
+                                }, function() {
+                                    handleLocationError(true, infoWindow, map.getCenter());
+                                });
+                                } else {
+                                // Browser doesn't support Geolocation
+                                handleLocationError(false, infoWindow, map.getCenter());
+                                }
+
+                                //Allow organizer to click on map to place a marker. 
+                                //TO-DO: Instead of displaying separate markers, draw path connecting each mouse click.
+                                map.addListener("click", function(e){
+                                    var marker = new google.maps.Marker({
+                                        position: e.latLng,
+                                        map: map,
+                                        draggable: false
+                                    });
+
+                                    //Click on marker to remove it in case of mis-clicks.
+                                    marker.addListener('click', function(f){
+                                        removeCoord(marker.position.lat(), marker.position.lng());
+                                        marker.setMap(null);
+                                    });
+
+                                    storeCoords(marker.position.lat(), marker.position.lng());
+                                });
+                            }
+
+                            function handleLocationError(browserHasGeolocation, infoWindow, pos) {
+                                infoWindow.setPosition(pos);
+                                infoWindow.setContent(browserHasGeolocation ?
+                                                    'Error: The Geolocation service failed.' :
+                                                    'Error: Your browser doesn\'t support geolocation.');
+                                infoWindow.open(map);
+                            }
+                            </script>
+                            <script async defer
+                            src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDQ00Z_l0PN-jiIyiUHTjn_jdnlY8HNn34&callback=initMap">
+                            </script>
+
+                            <!-- TO-DO: Send the values set by map.js to a Java servlet, to store the coordinate values in Event objects.-->
+                            <!-- Currently the input type is "text" for testing, but can be changed to "hidden" to not see the field on the page.-->
+                            <form action="/data">
+                                <input type="text" name="centerCoord" id="centerCoord" value="" />
+                                <input type="text" name="pathCoords" id="pathCoords" value="" />
+                                <button>Save</button>
+                            </form>
```

Comment:
This might not be the best solution, but I'm not sure how else to connect the inputs to Java to add into the Event objects. Any suggestions?